data/fields/sauna_water.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "sauna:water",
+    "type": "check",
+    "label": "Throwing water on the sauna stove"
```

Comment:
is both and `"dry": "Sauna without throwing water",` option needed? Seems to be tagging double single piece of info

Reply:
Well one doesn't normally throw water in steam sauna either.

Then there are normal hot saunas where throwing water is not allowed either.

It really depends.

This really makes a difference in the varm sauna.